signing-manifests/example-macapp.yml.tmpl

```diff
@@ -8,6 +8,7 @@ requestor: Ben Hearsum <bhearsum@mozilla.com>
 reason: test mac signing
 artifact-name: wireguard.tar.gz
 mac-entitlements-url: https://gist.githubusercontent.com/bhearsum/5b361f5834a467dd1a731810c2126ab9/raw/c3f68a4f9f8e5fcb5af8cd27aecf9d5bbd42c8e6/gistfile1.txt
+provisioning-profile-url: https://hg.mozilla.org/build/braindump/raw-file/tip/signing-related/firefoxvpn_developerid.provisionprofile
```

Comment:
Hm. I know this is an example, but can we pin this to a revision so it's reproducible rather than time-based?

Reply:
sure